src/main/java/com/google/firebase/auth/OidcProviderConfig.java

```diff
@@ -53,7 +53,8 @@ public String getIssuer() {
    * <p>Set the initial attributes of the new provider by calling various setter methods available
    * in this class.
    */
-  public static final class CreateRequest extends AuthProviderConfig.CreateRequest<CreateRequest> {
+  public static final class CreateRequest
+      extends AuthProviderConfig.AbstractCreateRequest<CreateRequest> {
```

Comment:
You can probably omit the `AuthProviderConfig.` now that the names are not ambiguous.

Reply:
Done.